decidim-core/app/cells/decidim/progress_bar/show.erb

```diff
@@ -7,7 +7,7 @@
   <%= content_tag :div, units_name_text, class: "progress-bar__units" if units_name %>
 
   <% if total != 0 %>
-    <div class="progress-bar" role="progressbar" tabindex="0" aria-valuenow="<%= percentage %>" aria-valuemin="0" aria-valuetext="<%= percentage %> percent" aria-valuemax="<%= total %>">
+    <div class="progress-bar" role="progressbar" tabindex="0" aria-label="<%= units_name.present? ? units_name_text : "progress" %>" aria-valuenow="<%= percentage %>" aria-valuemin="0" aria-valuetext="<%= percentage %> percent" aria-valuemax="<%= total %>">
```

Comment:
Shouldn't we translate the texts here ("progress" and "percent").

I notice that in the legacy design we used to have the percentage sign `%` instead of the literal text `percent` for the valuetext attribute. Either way is fine but I think it should be translated.

Reply:
Done